data/presets/emergency/fire_service_inlet.json

```
@@ -0,0 +1,32 @@
+{
+    "icon": "temaki-fire_hydrant",
+    "fields": [
+        "fire_mains",
+        "fire_service_inlet",
+        "couplings"
+    ],
+    "moreFields": [
+        "level",
+        "survey/date",
+        "fire_sprinkler"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "terms": [
+        "building hydrant inlet",
+        "charged riser",
+        "dry riser inlet",
+        "dry standpipe",
+        "fdc",
+        "fire service inlet",
+        "fire sprinkler inlet",
+        "riser inlet",
+        "wet riser inlet"
```

some more terms from https://en.wikipedia.org/wiki/Standpipe_(firefighting):

```suggestion
        "standpipe",
        "wet riser inlet",
        "wet standpipe"
```